D2Go/ObjectDetection/app/src/main/assets/classes.txt

```diff
@@ -0,0 +1,91 @@
+person
+bicycle
+car
+motorcycle
+airplane
+bus
+train
+truck
+boat
+traffic light
+fire hydrant
+street sign
+stop sign
+parking meter
+bench
+bird
+cat
+dog
+horse
+sheep
+cow
+elephant
+bear
+zebra
+giraffe
+hat
+backpack
+umbrella
+shoe
+eye glasses
+handbag
+tie
+suitcase
+frisbee
+skis
+snowboard
+sports ball
+kite
+baseball bat
+baseball glove
+skateboard
+surfboard
+tennis racket
+bottle
+plate
+wine glass
+cup
+fork
+knife
+spoon
+bowl
+banana
+apple
+sandwich
+orange
+broccoli
+carrot
+hot dog
+pizza
+donut
+cake
+chair
+couch
+potted plant
+bed
+mirror
+dining table
+window
+desk
+toilet
+door
+tv
+laptop
+mouse
+remote
+keyboard
+cell phone
+microwave
+oven
+toaster
+sink
+refrigerator
+blender
+book
+clock
+vase
+scissors
+teddy bear
+hair drier
+toothbrush
+hair brush
```

Comment:
I thought about it and normally I offer downloadable links for the models used in demo apps, but those models are much larger than 10MB. Thanks to D2Go, this model is just 7.5MB, and by adding it to the repo, maybe we're showing one of the strong points of D2Go :)

Reply:
Even if the artifact is only 7.5MB, I think it is still fairly large for a github repo, and once it's in here it will be in the history so even if we decide to remove it it will still be around.

I would say it's probably better to have the weights be downloaded from the internet

Reply:
After quantization, it's only 2.6MB now.. There's this [13.74MB model](https://github.com/pytorch/ios-demo-app/tree/master/HelloWorld/HelloWorld/HelloWorld/model) in the earlier HelloWorld demo and [3.43MB](https://github.com/pytorch/ios-demo-app/tree/master/PyTorchDemo/PyTorchDemo/ImageClassification/model) mobilenet model.. and a [44.72MB](https://github.com/pytorch/android-demo-app/tree/master/HelloWorldApp/app/src/main/assets) one.